test/AutoinstPartPlan_test.rb

```diff
@@ -53,44 +57,35 @@
          ]
         )
     end
-
-    it "ignoring not needed devices" do
-      target_map = YAML.load_file(target_map_clone)
-
-      expect(Yast::Storage).to receive(:GetTargetMap).and_return(target_map)
-      expect(subject.Read).to eq(true)
-      export = subject.Export.select { |d| d.key?("skip_list") }
-
-      expect(export[0]).to include("initialize" => true)
-      skip_list = export[0]["skip_list"]
-      expect(skip_list).to all(include("skip_key" => "device"))
-      expect(skip_list).to all(include("skip_value" => /\/dev\//))
-    end
   end
 
   describe "#Export" do
-    let(:target_map) { YAML.load_file(File.join(FIXTURES_PATH, "storage", "subvolumes.yml")) }
+
+    let(:exported) { subject.Export }
+    let(:sub_partitions) { exported.detect {|d| d["device"] == "/dev/sdd"}["partitions"] }
+    let(:subvolumes) { sub_partitions.first["subvolumes"].sort_by { |s| s["path"] } }
 
     before do
-      allow(Yast::Storage).to receive(:GetTargetMap).and_return(target_map)
       subject.Read
     end
 
     it "includes found subvolumes" do
-      exported = subject.Export
-      subvolumes = exported.first["partitions"].first["subvolumes"]
       expect(subvolumes).to eq([
-        { "path" => "@", "copy_on_write" => true},
-        { "path" => "home", "copy_on_write" => true },
-        { "path" => "var/log", "copy_on_write" => true },
-        { "path" => "var/lib/pgsql", "copy_on_write" => true },
-        { "path" => "myvol", "copy_on_write" => false },
+        {"path"=>"home", "copy_on_write"=>true},
```

Comment:
What about keeping the whitespaces? When we enable rubocop (in the future), it will complain about them.